source.js

```diff
@@ -701,6 +710,24 @@ var SVGtoPDF = function(doc, svg, x, y, options) {
       }
     }
 
+    /* Utilities for blend modes */
+    function getObjectStyles(obj) {
+      if (!obj.attributes.style) return {};
+
+      const styles = obj.attributes.style.split(';');
+      return styles.reduce((acc, style) => {
+        const [key, value] = style.split(':');

```

Comment:
shouldn't `key` and `value` be trimmed? in case `style = "foo : bar; baz : boom"`